src/index.js.flow

```diff
@@ -113,7 +113,7 @@ declare export function createChannel(connection: AmqpConnection|Promise<AmqpCon
 
 declare export function createManagedChannel(connection: AmqpManagedConnection): PublishChannel
 
-declare export function consumeFrom<C:ConsumeChannel> (channel: C): (config: ConsumeConfig, messageHandler: MessageHandler) => Promise<{}>
+declare export function consumeFrom<C:DuplexChannel> (channel: C): (config: ConsumeConfig, messageHandler: MessageHandler) => Promise<{}>
```

Comment:
What problem were you seeing here?  I'd like to understand it, since *conceptually*, consuming should only require a ConsumeChannel, and not a DuplexChannel.